lib/vdsm/tool/configurators/multipath.py

```diff
@@ -248,15 +40,9 @@
 
 def configure():
     """
-    Set up the multipath daemon configuration to the known and
-    supported state. The original configuration, if any, is saved
+    Set up the multipath daemon configuration and start the service.
     """
-    backup = fileUtils.backup_file(mpathconf.CONF_FILE)
-    if backup:
-        sys.stdout.write("Previous multipath.conf copied to %s\n" % backup)
```

Comment:
Another option to keep logging as is, is to pass a logger object to the mpathconf.configure_multiapthd(). We can use a standard logger object backed up
by sys.stdout, using simple log format, maybe only adding the logging level.

I think this should be fixed in vdsm-tool - use standard logger for all configurators, and stop using sys.stdout. But we cannot fix the entire world at the same time. We can start by adding a logger in this module, and later use the logger in all the configurators.

Reply:
I like this solution mainly because it entails make the tool use logging which is per se a nice and relatively easy improvement, that solves the issue naturally.

But I don't like the idea of having it mixed up (logging + stdout), it can be confusing. I'll create an issue with the proposal, and we can work on it in a dedicated PR.

Reply:
Issue: https://github.com/oVirt/vdsm/issues/258